config.toml

```diff
@@ -37,6 +37,11 @@ navbar_logo = true
     [markup.goldmark.renderer]
       unsafe = true
 
+  [markup.tableOfContents]
+    endLevel = 4
+#   ordered = false
+#   startLevel = 2
+    
```

Comment:
Setting `endLevel` increase the depth of the right hand side page ToC to show up to 4 headings, the change made for other PR review feedback. 

I kept the other YAML settings commented out in case it was tweaked again. Removed in commit 1271aa892abf3a8117ce656d4e30501a828eb669